ArduPlane/Attitude.cpp

```diff
@@ -209,7 +209,7 @@ float Plane::stabilize_pitch_get_pitch_out()
     const bool quadplane_in_frwd_transition = false;
 #endif
 
-    int32_t demanded_pitch = nav_pitch_cd + int32_t(g.pitch_trim * 100.0) + SRV_Channels::get_output_scaled(SRV_Channel::k_throttle) * g.kff_throttle_to_pitch;
+    demanded_pitch = nav_pitch_cd*0.01f + g.pitch_trim + SRV_Channels::get_output_scaled(SRV_Channel::k_throttle)*0.01f*g.kff_throttle_to_pitch;
```

Comment:
I think this also needs setting on 196 for the quadplane case. `use_fw_attitude_controllers` is not quite the same as `show_vtol_view` that we use for switching over logging, there maybe some overlap where this would be a change in behavior.

Reply:
bump on this^